controlplane/api/v1alpha1/rke2controlplane_types.go

```diff
@@ -102,6 +102,11 @@ type RKE2ServerConfig struct {
 	//+optional
 	CNI CNI `json:"cni,omitempty"`
 
+	// CNIMultusEnable enables multus as the first CNI plugin (default: false).
+	// This option will automatically overtake the CNI option as secondary CNI plugin.
```

Comment:
This statement is confusing at least to me. If we enable Multus CNI as a primary CNI via this field, why it is considered as secondary CNI plugin?

Reply:
Maybe we can create CNIPlugins field? In case we'll want to extend the list later

Reply:
Well, I think the godoc is unclear.  I might something like : 

```go
// This option will automatically make Multus primary CNI, and whatever mentioned in the cni field as secondary CNI plugin.
```

Would that work better or do you have a better formulation ?

Reply:
maybe:

`// This option will automatically make Multus a primary CNI, and the value, if specified in the CNI field, as a secondary CNI plugin.
`

Reply:
Sounds good, I just did that.